### DISCUSSION TITLE
How do I add external labels?

### DISCUSSION THREAD

**Discussion by Martmists-GH** [NationalSecurityAgency/ghidra#6898] [Q&A]
For example, I see a lot of code in this program referencing the address 0x06894000, which I'd like to label as VRAM_LCDC_E. How would I go about doing this for a list of pairs (address, label)?

**Comment by ryanmkurtz** [comment]
I think you are looking for `Symbol Table -> Right click on Imports -> Create External Location`

**Reply by Martmists-GH** [reply]
That doesn't seem to affect the Listing view or the Decompiler view, even after clearing the data and reassigning as `addr`.

**Comment by ryanmkurtz** [comment]
You should just be able to make a text file of the form:
```
label1 addr1
label2 addr2
...
```
and then from the `Script Manager`, run `ImportSymbolsScript.py`, giving it your text file.
